### PR TITLE
Topic/rem 18944 implement letting.tenancy agreement.create

### PR DESCRIPTION
**Pull request by gernotkogler**

### CODE REVIEW

File: masterdata_context.md [hunk 1]
```diff
@@ -365,6 +366,210 @@ data | hash |
 }
 ```
 
+### Masterdata.TenancyAgreement.Create
+
+This message is sent from an external message publisher to a GARAIO REM instance and allows to create a tenancy agreement. Set the recipient property in the headers, eg "grem_wincasa". Alwaays required attributes are noted in the remarks. Depending on the tenancy agreement template configuration, additional attributes may be required.
```
Comment (alexanderadam) [inline masterdata_context.md:371]:
```suggestion
This message is sent from an external message publisher to a GARAIO REM instance and allows to create a tenancy agreement. Set the recipient property in the headers, eg "grem_wincasa". Always required attributes are noted in the remarks. Depending on the tenancy agreement template configuration, additional attributes may be required.
```

File: masterdata_context.md [hunk 2]
```diff
@@ -365,6 +366,210 @@ data | hash |
...
+
+GARAIO REM replies with a standard Accepted / Rejected message containing the tenancyAgreementReference of the created tenancy agreement or reject reasons
```
Comment (alexanderadam) [inline masterdata_context.md:373]: Not important but I'd recommend in putting API terms or command in backticks for better readability.

```suggestion
GARAIO REM replies with a standard Accepted / Rejected message containing the `tenancyAgreementReference` of the created tenancy agreement or reject reasons
```

**Comment by alexanderadam** [comment]
Just a tiny typo but it looks good to me otherwise :+1:

**Comment by gernotkogler** [comment]
Thanks for the review, @alexanderadam

**Pull request merged**